sites/public/pages/listing.tsx

```diff
@@ -326,13 +302,29 @@ export default class extends Component<ListingProps> {
               <aside className="w-full static md:absolute md:right-0 md:w-1/3 md:top-0 sm:w-2/3 md:ml-2 h-full md:border border-gray-400 bg-white">
                 <div className="hidden md:block">
                   <ApplicationStatus listing={listing} />
-                  {openHouseEvents && <OpenHouseEventSection openHouseEvents={openHouseEvents} />}
+                  {lotteryResults && lotteryResults.url && (
+                    <section className="aside-block -mx-4 pt-0 md:mx-0 md:pt-4 text-center">
```

Comment:
Is this also something we should extract out to a component?

Reply:
uhhh, yeah missed that